### PR TITLE
Added new complete Task method which interacts with new complete task php script.

### PR DESCRIPTION
**Pull request by Adam746**
The method is very barebones at the moment.  It has no UI Implementation and doesn't get any return values from the script. It should be able to complete a task, though.

### CODE REVIEW

File: app/src/main/java/com/housemate/classes/Task.java [hunk 1]
```diff
@@ -8,10 +8,13 @@
 import com.fasterxml.jackson.annotation.PropertyAccessor;
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.housemate.activities.MainActivity;
 
+import java.net.MalformedURLException;
 import java.net.URL;
 import java.time.LocalDate;
 import java.util.ArrayList;
+import java.util.concurrent.ExecutionException;
```
Comment (Adam746) [inline app/src/main/java/com/housemate/classes/Task.java:17]: These can actually be removed.  They were automatically imported while I was writing the code but are no longer needed.

File: app/src/main/java/com/housemate/classes/Task.java [hunk 2]
```diff
@@ -8,10 +8,13 @@
 import com.fasterxml.jackson.annotation.PropertyAccessor;
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.housemate.activities.MainActivity;
 
+import java.net.MalformedURLException;
```
Comment (Adam746) [inline app/src/main/java/com/housemate/classes/Task.java:13]: These can actually be removed.  They were automatically imported while I was writing the code but are no longer needed.

File: app/src/main/java/com/housemate/classes/Task.java [hunk 3]
```diff
@@ -8,10 +8,13 @@
 import com.fasterxml.jackson.annotation.PropertyAccessor;
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.housemate.activities.MainActivity;
```
Comment (Adam746) [inline app/src/main/java/com/housemate/classes/Task.java:11]: These can actually be removed.  They were automatically imported while I was writing the code but are no longer needed.

**Review by KadeWern — APPROVED** [review]
Looks good to me! Nice!

**Pull request merged**